### PR TITLE
stability: add roll kill nemesis

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @jomenxiao @siddontang

### CODE REVIEW

File: stability-tester/suite/bank.go [hunk 1]
```diff
@@ -134,15 +138,28 @@ func (c *BankCase) initDB(ctx context.Context, db *sql.DB, id int) error {
...
 
 		for {
 			select {
 			case <-ticker.C:
-				c.verify(db, index)
+				err := c.verify(db, index)
+				if err != nil {
+					if time.Now().Sub(start) > defaultVerifyTimeout {
+						c.mu.Lock()
+						c.stop = true
```
Comment (siddontang) [inline stability-tester/suite/bank.go]: use atomic set/load

File: stability-tester/suite/bank.go [hunk 2]
```diff
@@ -154,7 +171,8 @@ func (c *BankCase) startVerify(ctx context.Context, db *sql.DB, index string) {
 func (c *BankCase) Execute(db *sql.DB, index int) error {
 	c.mu.RLock()
 	if c.stop {
-		return errors.New("bank stopped")
+		// too many log print in here if return error
+		return nil
```
Comment (siddontang) [inline stability-tester/suite/bank.go]: unlock here or use atomic load

**Comment by siddontang** [comment]
any update? @nolouch

File: stability-tester/suite/bank.go [hunk 3]
```diff
@@ -37,7 +38,7 @@ type BankCase struct {
 	cfg         *config.BankCaseConfig
 	concurrency int
 	wg          sync.WaitGroup
-	stop        bool
+	stopped     atomic.Value
```
Comment (siddontang) [inline stability-tester/suite/bank.go]: I prefer int64, not Value

**Comment by siddontang** [comment]
LGTM

PTAL @disksing

File: stability-tester/nemesis/rolling_update_nemesis.go [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+		if err != nil {
+			return errors.Trace(err)
+		}
+		log.Infof("RollingKillNemesis starts service %s", name)
+	}
+
+	return nil
+}
+
+func (n *RollingKillNemesis) GetSelector() Selector {
```
Comment (disksing) [inline stability-tester/nemesis/rolling_update_nemesis.go]: Need comment for exported method?

**Review by disksing — COMMENTED** [review]
The rest LGTM.

**Review by disksing — APPROVED** [review]

**Pull request merged**